site/.storybook/preview.jsx

```diff
@@ -24,12 +23,11 @@ const routerDecorator = (Story) => {
   )
 }
 
-addDecorator(routerDecorator)
+export const decorators = [themeProviderDecorator, routerDecorator]
```

Comment:
instead of `addDecorator`, we have to export a decorator array of decorators 🤷 : https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#removed-global-client-apis